### PR TITLE
Add queen mary college

### LINKED ISSUE #45: Physics Department, Queen Mary College, London,

### PR DESCRIPTION
**Pull request by eaculb**
Closes #45

### CODE REVIEW

File: db/institutions.yml [hunk 1]
```diff
@@ -262,4 +262,13 @@ nyu:
   name: New York University
   state: NY
   zip: 10003
+qmul:
+  aka: [QMUL, Queen Mary University of London, Queen Mary College, QMC]
+  city: London
+  country: UK
+  schools: {physastro: The School of Physics and Astronomy}
```
Comment (sbillinge) [inline db/institutions.yml]: schools and departments are now dicts of dicts.  please use other schools as an example

File: db/institutions.yml [hunk 2]
```diff
@@ -262,4 +262,13 @@ nyu:
...
+  departments: {cmmaterials: Centre for Condensed Matter and Materials Physics}
+  name: Queen Mary University of London
+  state: ''
```
Comment (sbillinge) [inline db/institutions.yml]: please a street address if you can find one.  The zip will be something like SW3 4ED

**Comment by eaculb** [comment]
Shoot. I'm not sure if I did this right. Tried to rebase and got "Current branch add_queen_mary_college is up to date." so I resolved via github but that doesn't look like a rebase.

**Comment by sbillinge** [comment]
I think this was a good rebase.  I am able to merge, but I will check.

**Pull request merged**